lib/asari.rb

```diff
@@ -23,9 +33,10 @@ def self.mode=(mode)
   attr_writer :search_domain
   attr_writer :aws_region
 
-  def initialize(search_domain=nil, aws_region=nil)
+  def initialize(search_domain = nil, aws_region = nil, api_version = nil)
```

Comment:
Would it be sensible to default `api_version` to the most recent to avoid changing the method's signature?